tests/Integration/Behaviour/Features/Scenario/Translation/legacy_system_fallback.feature

```diff
@@ -1,17 +0,0 @@
-# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml --name "Legacy Translation System"
```

Comment:
I'm not fan at all at removing these tests as "basically" these are the only ones that validates the expected behavior(s).

How about complete them instead?

Reply:
These tests have been replaced with integration and unit tests which are more thorough than the ones I removed -- except for the "install the module" part, which is another behavior (that's why I didn't delete the code that allows to test that even if currently no tests uses it).

See [ExternalModuleLegacySystemProviderTest.php](https://github.com/PrestaShop/PrestaShop/pull/13504/files#diff-5b5b57e15a62e14b4796f7ea9bb30053), and the other tests in `tests/Integration/PrestashopBundle/Translation` and `tests/Unit/PrestaShopBundle/Translation`